src/azul/plugins/metadata/hca/transform.py

```diff
@@ -71,6 +80,36 @@
 sample_types = api.CellLine, api.Organoid, api.SpecimenFromOrganism
 assert Sample.__args__ == sample_types  # since we can't use * in generic types
 
+pass_thru_uuid4: PassThrough[api.UUID4] = PassThrough()
+
+
+class OrganismAge(FieldType[JSON, str]):
+
+    def to_index(self, value: JSON) -> str:
+        try:
+            value, unit = value['value'], value['unit']
+        except TypeError:
+            value, unit = [None, None]
+        else:
+            assert '' not in (value, unit)
+        if value is None:
+            require(unit is None, 'A unit without value is not allowed', unit)
+            return NullableString.null_string
+        elif unit is None:
+            return value
+        else:
+            return value + ' ' + unit
```

Comment:
Since we use space as the joiner we need to ensure it doesn't occur in value or unit.